src/org/jitsi/impl/neomedia/rtcp/RTCPHeaderUtils.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.impl.neomedia.rtcp;
+
+import net.sf.fmj.media.rtp.*;
+
+/**
+ * Utility class that contains static methods for RTCP header manipulation.
+ *
+ * TODO maybe merge into the RTCPHeader class.
+ *
+ * @author George Politis
+ */
+public class RTCPHeaderUtils
+{
+    /**
+     * Gets the RTCP packet type.
+     *
+     * @param buf the byte buffer that contains the RTCP header.
+     * @param off the offset in the byte buffer where the RTCP header starts.
+     * @param len the number of bytes in buffer which constitute the actual
+     * data.
+     * @return the unsigned RTCP packet type, or -1 in case of an error.
+     */
+    public static int getPacketType(byte[] buf, int off, int len)
+    {
+        if (buf == null || Math.min(buf.length, len) < off + 2)
```

Comment:
off needs to be included in the check for buf.length, but not for len. E.g. the following is OK:
buf.length=10000, off=5000, len=10